backend/penndata/serializers.py

```diff
@@ -10,6 +12,9 @@
 )
 
 
+ValidatedData: TypeAlias = dict[str, Any]
```

Comment:
I feel like we don't want this type alias here (also I don't think you did above). Instead, if we are going to have this alias, we should have it in some common shared type file (same with the weird User type thing we are doing). 

But I think again, I'm wondering if we should be doing this "ad-hoc"-ly ourselves, and what existing large production type-safe Django projects do?